### DISCUSSION TITLE
Welcome to airplan Discussions!

### DISCUSSION THREAD

**Discussion by alexjbuck** [alexjbuck/airplan#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alexjbuck** [comment]
I'm a naval aviator and I like programming and building tools for folks. We discovered a bottleneck in some planning workflows and are building a tool to alleviate that!

**Comment by jccdev45** [comment]
Hello, I found this repo linked [here](https://www.mygoodfirstissue.com/) in my search for a good starting point to begin contributing to the code of other devs. Cheers, thanks for the opportunity!
